Sources/DistributedActors/Pattern/WorkerPool.swift

```diff
@@ -44,19 +54,79 @@ public struct WorkerPool<Message: ActorMessage> {
         /// The worker pool will terminate itself if all of its static workers have terminated.
         /// You may death-watch the worker pool in order to react to this situation, e.g. by spawning a replacement pool,
         /// or gracefully shutting down your application.
-        case `static`([_ActorRef<Message>])
+        case `static`([Worker])
```

Comment:
Hmm... we might need to be careful here and _not_ store the `[Worker]` as they'd never be able to be removed by terminating then 🤔

Reply:
Not storing the `[Worker]` or `Selector`.